packages/lesswrong/server/resolvers/commentResolvers.ts

```diff
@@ -4,18 +4,21 @@ import { accessFilterSingle } from '../../lib/utils/schemaUtils';
 
 const specificResolvers = {
   Mutation: {
-    moderateComment(root, { commentId, deleted, deletedPublic, deletedReason}, context) {
+    async moderateComment(root, { commentId, deleted, deletedPublic, deletedReason}, context: ResolverContext) {
+      const {currentUser} = context;
       const comment = context.Comments.findOne(commentId)
+      if (!comment) throw new Error("Invalid commentId");
       const post = context.Posts.findOne(comment.postId)
-
-      if (Users.canModeratePost(context.currentUser, post)) {
+      if (!post) throw new Error("Cannot find post");
```

Comment:
kinda weird for these to give two different Errors. Not a big deal but seems better to fix.

Reply:
I don't think I follow. Why would it bad for the error messages to differ here?

Reply:
It seemed generally better on the margin for error messages to be consistent. I'm not sure there's a concrete failure mode I'm imagining, just a general "small inconsistencies in code styling make it slightly harder to parse things. This isn't so huge an effect size that you should stress about it, but in a single PR all things being equal it seemed better to be consistent"

But, again, not a big deal, and to be clear, not a blocking request.